packages/neuron-wallet/src/models/chain/script.ts

```diff
@@ -0,0 +1,51 @@
+import { scriptToHash } from "@nervosnetwork/ckb-sdk-utils"
+import HexUtils from 'utils/hex'
+import TypeChecker from "utils/type-checker"
+
+export enum ScriptHashType {
+  Data = 'data',
+  Type = 'type',
+}
+
+export default class Script {
+  constructor(
+    public codeHash: string,
+    public args: string,
+    public hashType: ScriptHashType
+  ) {
+    this.args = args
+    this.codeHash = codeHash
+    this.hashType = hashType
```

Comment:
See blow. It looks more clearer to explicitly declare properties when we're not using constructor assignment shorthand (empty `{}`).

Reply:
I forgot to delete those lines.